hack/verify-contextual-logging.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o pipefail
+# Don't set errexit because logcheck's messages are in stderr
+
+SHOW=${1:-}
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# All subdirectories under ./pkg containing *.go files, _except_ in pkg/apis.
+# logcheck doesn't like that our main module doesn't reference pkg/apis.
+PKGS=$(cd "${SCRIPT_ROOT}"; find ./pkg -type f -name "*.go" -not -path "./pkg/apis/*" | sed -r 's|/[^/]+$||' | sort | uniq)
+
+
+errs=$(logcheck -check-contextual ${PKGS} 2>&1)
```

Comment:
I'm able to do `logcheck -check-contextual ./...` - what problems were you seeing?

Reply:
I actually wasn't aware that `./...` could be used; I'm going to switch over to that because it's much cleaner :)